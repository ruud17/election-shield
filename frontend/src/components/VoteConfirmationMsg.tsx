import { FC } from "react";
import { Row, Alert, Image } from "react-bootstrap";
import SuccessVoteIcon from "../images/success_vote.svg";

const VoteConfirmationMsg: FC = () => {
  return (
    <div className="success-vote">
      <Row>
        <Image src={SuccessVoteIcon} />
      </Row>
      <Row>
        <Alert variant="success">
          <Alert.Heading>You have successfully voted</Alert.Heading>
          <hr />
          <p className="mb-0">Thank you!</p>
        </Alert>
      </Row>
    </div>
  );
};

export default VoteConfirmationMsg;
