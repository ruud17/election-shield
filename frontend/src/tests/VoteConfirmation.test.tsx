import renderer from "react-test-renderer";
import VoteConfirmationMsg from "../components/VoteConfirmationMsg";

test("renders correctly", () => {
  const tree = renderer.create(<VoteConfirmationMsg />).toJSON();

  expect(tree).toMatchSnapshot();
});
