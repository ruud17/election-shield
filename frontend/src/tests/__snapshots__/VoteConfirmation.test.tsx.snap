// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="success-vote"
>
  <div
    className="row"
  >
    <img
      className=""
      src="success_vote.svg"
    />
  </div>
  <div
    className="row"
  >
    <div
      className="fade alert alert-success show"
      role="alert"
    >
      <div
        className="alert-heading h4"
      >
        You have successfully voted
      </div>
      <hr />
      <p
        className="mb-0"
      >
        Thank you!
      </p>
    </div>
  </div>
</div>
`;
